docs/concepts/crds/backupconfiguration.md

```diff
@@ -0,0 +1,199 @@
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `serviceAccountName`           | Name of the `ServiceAccount` to use for backup job. Stash sidecar will use the same `ServiceAccount` as the target.                                                                                                                      |
+| `nodeSelector`                 | Selector which must be true for backup job pod to fit on a node.                                                                                                                                                                         |
+| `automountServiceAccountToken` | Indicates whether a service account token should be automatically mounted into the backup pod.                                                                                                                                           |
+| `nodeName`                     | NodeName is used to request to schedule backup job's pod onto a specific node.                                                                                                                                                           |
+| `securityContext`              | Security options that backup job's pod should run with. For more details, please visit [here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).                                                               |
+| `imagePullSecrets`             | A list of secret names in the same namespace that will be used to pull image from private Docker registry. For more details, please visit [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). |
+| `affinity`                     | Affinity and anti-affinity to schedule backup job's pod in the desired node. For more details, please visit [here](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).                       |
+| `schedulerName`                | Name of the scheduler that should dispatch the backup job.                                                                                                                                                                               |
+| `tolerations`                  | Taints and Tolerations to ensure that backup job's pod is not scheduled in inappropriate nodes. For more details about `toleration`, please visit [here](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).       |
+| `priorityClassName`            | Indicates the backup job pod's priority class. For more details, please visit [here](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/).                                                                        |
```

Comment:
I worry about these links to Kubernetes docs site. They break their links every time they update the docs without much care.